### DISCUSSION TITLE
production.ERROR: Could not verify password security: cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.pwnedpasswords.com/range/acb52

### DISCUSSION THREAD

**Discussion by TheFork001** [firefly-iii/firefly-iii#8410] [General]
SSL is enabled and it does work, the db does connect but when checking the password security the app fails, any ideas? Am I missing something?

[2024-01-17 14:54:38] production.ERROR: Could not verify password security: cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.pwnedpasswords.com/range/acb52

**Comment by JC5** [comment]
Mmm, no clue. This has to do with the "verify password security" checkbox when registering. You can disable it without any ill effect if you want.

**Reply by TheFork001** [reply]
do you know what file hold the disable featuer?

**Reply by JC5** [reply]
It's a checkbox on the registration screen.

**Reply by TheFork001** [reply]
No right, I mean id there a way to disable this globally to not show that check box.

**Reply by TheFork001** [reply]
On a quick side note, to configure the check for updates where it reaches out, is that something we also need to configure in the .env file? if so where? because when I click it it says ran into a guzzle error

**Reply by JC5** [reply]
The update check is in the /admin settings, or when you first get the question to check for updates there is an option to disable it. There are currently no plans to allow you to remove the checkbox entirely.

**Reply by TheFork001** [reply]
Got it so I want to make sure I'm understanding. The is no option currently to disable that check box, I will continue trying to troubleshoot it on my end. And as far as the updates go, I know where it is but there is no configuration needed for that right? If so then I may need to check a few things on my end that could solve both issues. So to clarify, check for updates should be working straight out of the box right?

**Reply by JC5** [reply]
It will not check for updates untill you allow it, but it should work out of the box!

**Reply by TheFork001** [reply]
I fixed it. I was missing a curl file, I got everything going will update the iis show and tell guide!

**Comment by TheFork001** [comment]
Closing, user error. solution is here: https://github.com/orgs/firefly-iii/discussions/8400
